content/en/02-installing-pixie/03-install-guides/02-self-hosted-pixie.md

```diff
@@ -76,7 +76,7 @@ kustomize build k8s/cloud/public/ | kubectl apply -f - --namespace=plc
 kubectl get pods --all-namespaces -o wide
 ```
 
-### Set up DNS
+### Set up DNS for this cluster
```

Comment:
I think this is sort of unclear. It's updating the DNS for the user's local machine to point to the user's cluster, but this phrase makes me think we're configuring the cluster's DNS somehow.

Reply:
good point